### PR TITLE
php: bump version

### PR DESCRIPTION
**Pull request by kenmays**
Bumped to PHP 7.4.30 and PHP 8.1.9.
Unit tested on Haiku hrev56350.
Closes: https://github.com/haikuports/haikuports/issues/1588
Closes: https://github.com/haikuports/haikuports/issues/7076

### CODE REVIEW

File: dev-lang/php/php-7.4.30.recipe [hunk 1]
```diff
@@ -50,44 +50,41 @@ REQUIRES="
...
 	lib:libpcre$secondaryArchSuffix
 	lib:libpng16$secondaryArchSuffix
 	lib:libsqlite3$secondaryArchSuffix
 	lib:libssl$secondaryArchSuffix
-	lib:libwebp$secondaryArchSuffix
 	lib:libxml2$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
-	lib:libzip$secondaryArchSuffix
+	lib:libwebp$secondaryArchSuffix
+	lib:libonig$secondaryArchSuffix
```
Comment (augiedoggie) [inline dev-lang/php/php-7.4.30.recipe:60]: Why were these moved? They need to be alphabetized.

File: dev-lang/php/php-7.4.30.recipe [hunk 2]
```diff
@@ -50,44 +50,41 @@ REQUIRES="
...
-	devel:libpq$secondaryArchSuffix
 	devel:libsqlite3$secondaryArchSuffix
 	devel:libssl$secondaryArchSuffix
-	devel:libwebp$secondaryArchSuffix
 	devel:libxml2$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
-	devel:libzip$secondaryArchSuffix
+	devel:libwebp$secondaryArchSuffix
+	devel:libonig$secondaryArchSuffix
+	devel:libpq$secondaryArchSuffix
```
Comment (augiedoggie) [inline dev-lang/php/php-7.4.30.recipe:80]: The gd/onig/pq/webp entries need to be alphabetized as they were before.

File: dev-lang/php/php8-8.1.9.recipe [hunk 3]
```diff
@@ -85,7 +85,7 @@ BUILD_REQUIRES="
 	devel:libpcre2_8$secondaryArchSuffix
 	devel:libpng16$secondaryArchSuffix
 	devel:libpq$secondaryArchSuffix
-	devel:libreadline$secondaryArchSuffix >= 8
+	devel:libreadline$secondaryArchSuffix
```
Comment (augiedoggie) [inline dev-lang/php/php8-8.1.9.recipe:88]: Why was the version requirement removed?

**Review by augiedoggie — CHANGES_REQUESTED** [review]
It seems like the patches were needlessly regenerated instead of just being renamed.  Did they not apply properly?

File: dev-lang/php/php-7.4.30.recipe (hunk 1 above)
Comment (kenmays) [reply dev-lang/php/php-7.4.30.recipe:60]: Patches were regenerated after a successful build.

File: dev-lang/php/php-7.4.30.recipe (hunk 2 above)
Comment (kenmays) [reply dev-lang/php/php-7.4.30.recipe]: Fixed

File: dev-lang/php/php-7.4.30.recipe (hunk 1 above)
Comment (kenmays) [reply dev-lang/php/php-7.4.30.recipe]: Fixed

**Comment by augiedoggie** [comment]
Why was zip support removed?  It would help to have that information in the commit message or pull request comments.

I don't use php on Haiku so I don't have an opinion about which modules are enabled. Someone else can decide that.

File: dev-lang/php/php-7.4.30.recipe [hunk 4]
```diff
@@ -106,7 +105,6 @@ BUILD()
 		--sysconfdir=$settingsDir/php7 \
 		--with-config-file-path=$settingsDir/php7 \
 		--with-config-file-scan-dir=$settingsDir/php7 \
-		--with-zip=shared \
```
Comment (augiedoggie) [inline dev-lang/php/php-7.4.30.recipe:109]: This should also be restored to enable zip support.

**Review by augiedoggie — APPROVED** [review]
